pyspades/packet.pyx

```diff
@@ -76,19 +79,24 @@ def register_packet(loader=None, server=True, client=True, extension=None):
         # Since this library tries to work as both a Server and Client
         # implementation, both are always evaluated
 
-        if client:
-            if cls.id in _client_loaders:
-                msg = ("Tried to register client packet '{!r}', but a packet "
-                       "with that id ({}) already exists.".format(cls, cls.id))
-                raise KeyError(msg)
-            _client_loaders[cls.id] = cls
-
-        if server:
-            if cls.id in _server_loaders:
-                msg = ("Tried to register server packet '{!r}', but a packet "
-                       "with that id ({}) already exists.".format(cls, cls.id))
-                raise KeyError(msg)
-            _server_loaders[cls.id] = cls
+        for version in range(cls.since_version, cls.until_version, 1):
```

Comment:
I'd prefer if instead of building the lookup table for all versions in between at initialization, a loader dictionary for a specific version was requested via parameters (and then cached).